pkg/controller/v1beta1/inferenceservice/controller.go

```diff
@@ -254,8 +259,10 @@ func (r *InferenceServiceReconciler) Reconcile(ctx context.Context, req ctrl.Req
 		if isvc.Spec.Explainer != nil {
 			componentList = append(componentList, v1beta1.ExplainerComponent)
 		}
-		isvc.Status.PropagateCrossComponentStatus(componentList, v1beta1.RoutesReady)
-		isvc.Status.PropagateCrossComponentStatus(componentList, v1beta1.LatestDeploymentReady)
+		if !forceStopRuntime {
```

Comment:
Without the conditional, wouldn't the resulting status be accurate?

Reply:
Add comment explaining that the routes and deployments are removed / never made , so the conditions are not needed